docs/docs-content/vertex/system-management/account-management/credentials.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_label: "User Credentials"
+title: "User Credentials"
+description: "Update and manage the user credentials"
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 20
+tags: ["vertex", "management", "account", "credentials"]
+keywords: ["self-hosted", "vertex"]
+---
+
+You can manage the credentials of the admin user by logging in to the system console. You can also enable passwordless login for the admin user by adding passkeys. 
+
+Use the sections below to learn how to manage user credentials, such as changing the admin password, and enabling passwordless login by adding passkeys.
```

Comment:
```suggestion
Use the sections below to learn how to manage user credentials.
```

Reply:
There was a bit of repetition here from the previous paragraph.

Reply:
Good suggestion ✔️